pkg/auth/common/login_handler.go

```diff
@@ -113,6 +113,14 @@ func WithOrgID(orgID string) LoginOption {
 	}
 }
 
+// WithClientID causes the login with given ClientID.
+func WithClientID(clientID string) LoginOption {
+	return func(h *TanzuLoginHandler) error {
+		h.clientID = clientID
```

Comment:
I see in @vuil PR https://github.com/vmware-tanzu/tanzu-cli/pull/819/files#diff-c2506254f4d5d93b4c84bfa32c18782a79fbca5c788f38444419c9af5060cca1R130 that he also sets the `h.oauthConfig.ClientID` field here.

Reply:
This needs to be updated as part of rebase after #819 gets merged because the same function is implemented there as well.